### ISSUE TITLE
Revise the eye process'es "Algorithm" view

### ISSUE THREAD

**Issue by papr** [pupil-labs/pupil#877]
As seen in #851, the algorithm view is dependent on the `Flip Image` and positions its main elements sub-optimally. I also would like to discuss the option to implement the algorithm view with opengl instead of drawing it into the image buffer.

**Comment by willpatera** [comment]
@papr I don't think this will make the current milestone, but should be done for the next release/milestone.
